docs/faq/proof-of-stake/pos-general-faq.md

```diff
@@ -0,0 +1,123 @@
+
+The PoS system rewards participants for their staking. Every Decred block has a PoS reward component that will be distributed to every ticket chosen to vote in that block. The system is designed in such a way that the average time for the ticket to be selected is 28 days. If a ticket hasn't been selected after 40960 blocks (about 4 months) it will be revoked by the system. The DCR used to purchase the ticket will be returned to the ticket owner.
+
+Another interesting feature this PoS system allows for is the ability to vote on an agenda. Because the block validation can be considered a vote, by adding extra votebit components to the ticket the system can track and count votes over a series of blocks. These votes can be about anything.
+
+In the Decred Proof-of-Stake system anyone holding DCR can participate by buying a ticket.
+
+---
+
+#### **2. What are tickets?**
+
+A ticket is a token you buy to participate in the PoS system. You can buy tickets through the command line interface of dcrctl, or through a GUI like Paymetheus[^1] or Decrediton. Whenever you buy a ticket, you pay the current ticket price and a ticket fee using your DCR. Once your ticket is mined it will move from the current 'allmempool' to the 'immature' ticket pool. After 256 blocks (about 22 hours) it then will mature and go into the live ticket pool, and is ready to be chosen to validate a block.
+
+Remember, your wallet has to be online and unlocked for your ticket to cast a valid vote on a block when it is selected. If you don't want to leave your wallet unlocked and online, have a look at a PoS stake pool.
+
+---
+
+#### **3. What is ticket price?**
+
+The price for tickets is determined by an algorithm. This algorithm aims to keep the ticket pool size, which is the total amount of tickets in the PoS system ticket pool, at around the target. The target is set to 40960 tickets. The ticket price will go up or down according to the demand for tickets.
+
+Every 144 blocks the algorithm adjusts the ticket price. This is called a buying window. Each block can contain 20 newly bought tickets. This means that in every buying window a maximum of 2880 tickets can be added to the PoS system ticket pool.
+
+---
+
+#### **4. What are fees?**
+
+The PoS system utilizes two types of fees, a txfee and a ticketfee.
+	
+The txfee is a fee you pay the network for handling the transaction to buy your ticket. This fee is at default set to a low amount (0.01 DCR/kB) and should not be changed.
+The ticketfee is the fee you pay if your ticket gets mined. This fee is an incentive for PoW miners to mine your ticket and add it to the ticket pool.
```

Comment:
here i would mention that the "ticket fee" is also a transaction fee, but the "ticket fee" that you configure in the software only applies to ticket purchase transactions.